phh-prop-handler.sh

```diff
@@ -257,3 +257,17 @@ if [ "$1" == "persist.bluetooth.system_audio_hal.enabled" ]; then
     restartAudio
     exit
 fi
+
+if [ "$1" == "persist.sys.phh.force_display_5g" ];then
+    if [[ "$prop_value" != "0" && "$prop_value" != "1" ]]; then
+        exit 1
+    fi
+
+    if [[ "$prop_value" == 1 ]];then
+        resetprop_phh ro.telephony.default_network 33,33
+    else
+        resetprop_phh ro.telephony.default_network ${ro.vendor.radio.default_network}
```

Comment:
Not all devices have that prop though. Some will directly set it in build.prop, some will set it from some shell script, some will set it from rc script, ...
I'd rather have this option require reboot, and in rw-system.sh have `if [ "$(getprop persist...)" = 1 ];then resetprop_phh ...;fi`

Reply:
Makes sense. Will do. I need to update TrebleApp toggle summary then.